script.js

```diff
@@ -0,0 +1,212 @@
+    editWrapForm.classList.add('hidden');
+};
+
+const newCard = ({id, description, status, date, lastEditDate}) => {
+    const cardDiv = document.createElement('div');
+    cardDiv.setAttribute('class','card_task');
+    cardDiv.setAttribute('draggable', 'true');
+    cardDiv.setAttribute('data-id', id);
+    const content = `
+            <h2>${description}</h2>
+            <p>${date}</p>
+            <div class='card-content'>
+            <p>Ostatnia edycja: </p>
+            <p>${lastEditDate}</p>
+            </div>
+    `
+    const cardEdit = document.createElement('div');
+    const editBtn = document.createElement('button');
+    const deleteBtn = document.createElement('button');
+
+    cardEdit.classList.add('card_edit');
+    editBtn.classList.add('edit_btn');
+    deleteBtn.classList.add('delete_btn');
+ 
+
+    cardDiv.innerHTML = content;
+    cardEdit.appendChild(editBtn);
+    cardEdit.appendChild(deleteBtn);
+    cardDiv.appendChild(cardEdit);
+
+    editBtn.addEventListener('click', () => {
+        editWrapForm.classList.remove('hidden');
+        editWrapForm.classList.add('visible');
+        const tasksDB = getDB();
+        const filteredTaskDB = tasksDB.filter(el => el.id === id);
+        editTaskInput.value = filteredTaskDB[0].description;
+        editSelectOption.value = filteredTaskDB[0].status;
+        idInput.value = id;
+    });
+
+    deleteBtn.addEventListener('click', () => {
+        let tasksDB = getDB();
+        const filteredTaskDB = tasksDB.filter(el => el.id !== id);
+        setDB(filteredTaskDB);
+       refreshView(filteredTaskDB);
+    });
+
+    closeTaskBtn.addEventListener('click', closeForm);
+
+    return cardDiv;
+};
+
+const listOfTasks = (list, container) => {
+    const numOfTasks = list.length;
+    container.previousElementSibling.querySelector(".counter").textContent = numOfTasks;
+    container.innerHTML = '';
+    list.forEach( (el) => {
+        const card = newCard(el);
+        container.appendChild(card);
+    });
+};
+
+
+const createNewDate = () => {
+    return new Intl.DateTimeFormat(
+        window.navigator.language, 
+        { dateStyle: 'full', timeStyle: 'medium' }
+        ).format(new Date());
+
+    };
+    
+newTaskForm.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const tasksDB = getDB();
+    tasksDB.push({
+        id: new Date().getTime().toString(),
+        description: newTaskInput.value,
+        status: selectOption.value,
+        date: new Date().toISOString(),
+        lastEditDate: new Date().toISOString()
+    });
+    setDB(tasksDB);
+    const {toDoStatus, inProgressStatus, doneStatus} = filteredStatus(tasksDB);
+    if(selectOption.value === 'to_do'){
+        listOfTasks(toDoStatus, toDoContainer);
+    }else if(selectOption.value === 'in_progress') {
+        listOfTasks(inProgressStatus, inProgressContainer);
+    }else if(selectOption.value === 'done'){
+        listOfTasks(doneStatus, doneContainer);
+    }
```

Comment:
`switch` would be more clear 

and statuses could be kept in consts

Reply:
Done 👍